Update server-y.adoc, server-major-upgrade-uyuni.adoc and .changelog (#1909)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/server-y.adoc

This does not apply if you are logged in from a remote machine (unless you are running a VNC session with GNOME).
====

> [IMPORTANT]

Reviewer: I think this could be relevant for `modules/installation-and-upgrade/pages/server-major-upgrade-uyuni.adoc`?

(The .adoc you are changing is only for SUSE Manager).

Can you also add a changelog entry at `.changelog` at the root of the repo? :-)

Reviewer: Oh, and clarify if we need to backport this.

I'd say yes, so you need to mark all three checkboxes at the description.

Author: I can update .changelog. However there is no hardware requirements for uyuni, this should be created first.

Reviewer: We do have them:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/installation-and-upgrade/uyuni-install-requirements.html

So there should be a `uyuni-install-requirements.adoc`, if I am not wrong.

Author: I modified the PR accordingly.

====

> Before the upgrade, you need to ensure that the storage requirements are still valid. Refer to hardware-requirements.adoc. Migration can fill root partition if there is not enough space available due to the service pack migration and download of new rpms. It is the same for the /var/lib/pgsql, when upgrading postgresql, it takes a copy of the old database, be sure to have at least enough space available to cope with a copy of the database.

Reviewer: ```suggestion
Before the upgrade, you need to ensure that the storage requirements are still valid. Refer to xref:hardware-requirements.adoc[]. Migration can fill root partition if there is not enough space available due to the service pack migration and download of new RPMs. It is the same for the `/var/lib/pgsql`, when upgrading PostgreSQL, it takes a copy of the old database, be sure to have at least enough space available to cope with a copy of the database.
```

> Before the upgrade, you need to ensure that the storage requirements are still valid. Refer to xref:hardware-requirements.adoc[]. Migration can fill root partition if there is not enough space available due to the service pack migration and download of new RPMs. It is the same for the `/var/lib/pgsql`, when upgrading PostgreSQL, it takes a copy of the old database, be sure to have at least enough space available to cope with a copy of the database.

Reviewer 2: ```suggestion
Before the upgrade, ensure that storage requirements are met.
For more information, see xref:hardware-requirements.adoc[].
The migration procedure can fill the root partition if there is not enough space available due to the service pack migration and the download of new software packages.
It is the same for the [path]``/var/lib/pgsql`` when upgrading PostgreSQL.
It takes a copy of the old database, thus be sure to have at least enough space available to cope with a copy of the database.
```

====

> 

Reviewer 2: ```suggestion
```
